src/ert/job_queue/job_queue_node.py

```diff
@@ -110,7 +110,7 @@ def free(self) -> None:
 
     def __str__(self) -> str:
         return (
-            f"JobNode: Name:{self.run_arg.job_name}, Status: {self.queue_status}, "
+            f"ForwardModelNode: Name:{self.run_arg.job_name}, Status: {self.queue_status}, "
```

Comment:
Maybe go all the way, ForwardModelStepNode.

Reply:
I can change that, but I do not want to change more in this file as it will hopefully be deleted soon anyways.

Reply:
> I can change that, but I do not want to change more in this file as it will hopefully be deleted soon anyways.

Well not so soon (SLURM .....)

Reply:
Should I update all instances of `Job` to `ForwardModelStep`, or leave it be?